stacks/core/README.md

```diff
@@ -13,19 +13,82 @@ If you don't have a domain or can't configure the main DNS server, you can confi
 
 On Windows, the hosts file is usually located under `c:\Windows\System32\Drivers\etc\hosts`. On Unix systems, it is commonly found at `/etc/hosts`. Regardless of you operating system, administrator or super user privileges will be required.
 
-### Deploy: stage 1
+### Deploy
 
-Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stack/env.d`.
+The `core` stack uses multiple compose files to setup the services and reach the desired state. Definitions common to both development and production settings are grouped in the `base` directory and specific ones are grouped in the `dev` and `prod` directories. For example, when configuring `traefik` as service's load balancer, you can deploy its development definition to avoid exposing services without need. Also, it can be necessary to deploy the services into multiple machines, which could be done by using the `multi-node` definitions.
```

Comment:
The `core` stack uses multiple compose files to setup the services and reach the desired state. **Common definitions** to both development and production settings are grouped in the `base` directory and specific ones are grouped in the `dev` and `prod` directories. For example, when configuring `traefik` as service's load balancer, you can deploy its development definition to avoid exposing services without need. Also, it can be necessary to deploy the services into multiple machines, which could be done by using the `multi-node` definitions.

Reply:
Done.